Fill symfony/validator bridge documentation (#145)
yokai-php/batch-src: docs/docs/bridges/symfony-validator.rst

Bridge with ``symfony/validator``
============================================================

> See `official documentation <https://symfony.com/doc/current/validation.html>`__ on Symfony's website.

Reviewer: ```suggestion
Refer to the  `official documentation <https://symfony.com/doc/current/validation.html>`__ on Symfony's website.
```

Skip invalid item processor
------------------------------------------------------------

> | This item processor will validate every item, and throw an exception if at least one violation is raised.

Reviewer: ```suggestion
| This item processor will validate each item, and throws an exception if at least one violation is raised.
```

> | The exception is caught by ``ItemJob``, and the item will be skipped.

Reviewer: ```suggestion
| The exception is caught by ``ItemJob``, causing the item to be skipped.
```
